actioncable/lib/action_cable/channel/streams.rb

```diff
@@ -124,6 +124,18 @@ def stop_all_streams
         end.clear
       end
 
+      # Calls stream_for if record is present, otherwise calls reject.
+      # This method is intended to be called when you're looking
+      # for a record based on a parameter, if its found it will start
+      # streaming. If the record is nil then it will reject the connection.
```

Comment:
I think the code is already declarative enough. The comment is no longer needed.

Reply:
I am ok to remove it when I get another comment from someone from core/committer team(who has access to merge this PR)

Reply:
That's reasonable.